apps/sms/views/conversation/js/conversation.js

```diff
@@ -642,6 +646,7 @@ var ConversationView = {
     if (Navigation.isCurrentPanel('thread-list')) {
       this.container.textContent = '';
       this.cleanFields();
+      // TODO: Will need to replace with Conversation service method.
```

Comment:
Same question about how we deal with currentId (https://github.com/mozilla-b2g/gaia/pull/30798#discussion_r33837856)